src/app/organizations/manage/policies.component.ts

```diff
@@ -149,4 +155,15 @@ export class PoliciesComponent implements OnInit {
         } catch { }
         this.enterpriseTokenPromise = null;
     }
+
+    private updatePolicyVisibility(organization: Organization) {
```

Comment:
Would it be cleaner to just add a `display` boolean of some kind to the policy list? Something like:

```
        this.policies = [
            {
                name: i18nService.t('twoStepLogin'),
                description: i18nService.t('twoStepLoginPolicyDesc'),
                type: PolicyType.TwoFactorAuthentication,
                enabled: false,
                display: true,
            },
            {
                name: i18nService.t('masterPass'),
                description: i18nService.t('masterPassPolicyDesc'),
                type: PolicyType.MasterPassword,
                enabled: false,
                display: true,
            },
            {
                name: i18nService.t('passwordGenerator'),
                description: i18nService.t('passwordGeneratorPolicyDesc'),
                type: PolicyType.PasswordGenerator,
                enabled: false,
                display: true,
            },
            {
                name: i18nService.t('onlyOrg'),
                description: i18nService.t('onlyOrgDesc'),
                type: PolicyType.OnlyOrg,
                enabled: false,
                display: true,
            },
            {
                name: i18nService.t('requireSso'),
                description: i18nService.t('requireSsoPolicyDesc'),
                type: PolicyType.OnlyOrg,
                enabled: false,
                display: organization.userSso,
            },
        ];
```

Reply:
ohh I like this a lot - moving some things around and will update!

Reply:
Updated 👍